FlightOps/manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
```

Comment:
```suggestion
import sys

def main():
```

espaco desnecessario